source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,287 @@
+.. _swupd-3rd-party:
+
+swupd 3rd-party 
+###############
+
+Upstream Clear Linux offers a plethora of `bundles`_ to choose from.  
+
+To provide you access to even more software outside of the distro, 
+the :ref:`mixer` and :ref:`swupd-guide` tools have been updated to support 3rd-party
+bundles starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.
+
+And to consume and manage these bundles, we have added a `3rd-party` 
+subcommand to :command:`swupd`.
+
+Follow this guide to see how to build an example 3rd-party bundle, set up a web 
+server to host it, install and manage the bundle on a client system. 
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisite
+*************
+
+* Familiarity with :command:`mixer`
```

Comment:
What do you think about using one of these "levels"? basic, intermediate, or advanced

I think this would be stronger as:
"Intermediate: Some experience with :command:`mixer`
"Intermediate: Some experience with :command:`swupd`

We could also create a parent document (higher in tree) where we define the skill levels. 
That way, here we'd just use "skill-level: intermediate: "
I think creating these tags in our docs would help.

Reply:
@mvincerx will create an issue for this. Don't let this block.